teams/teams-ps/teams/Remove-CsTeamTemplate.md

```diff
@@ -232,33 +231,34 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## NOTES
 
+ALIASES
+
 COMPLEX PARAMETER PROPERTIES
 
-To create the parameters described below, construct a hash table containing the appropriate properties.

```

Comment:
I think I have originally misformatted the complex parameter properties. I have not manually tweaked them, it's just the visual changes